modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -7,4 +7,6 @@ Bundle-Vendor: Nuxeo
 Nuxeo-Component: OSGI-INF/test-document-routing-operations-contrib.xml,
   OSGI-INF/test-document-routing-operation-chains-contrib.xml,
   OSGI-INF/test-document-routing-type-chains-contrib.xml,
-  OSGI-INF/test-document-routing-listeners.xml
+  OSGI-INF/test-document-routing-listeners.xml,
+  OSGI-INF/test-reassign-delegate-listener.xml
+  
```

Comment:
There must be a proper newline (and no lonely space) at the end of the MANIFEST.MF.